application/clicommands/JobsCommand.php

```diff
@@ -108,28 +120,49 @@ public function runAction()
     }
 
     /**
-     * Fetch jobs from disk
+     * Fetch job schedules from database
      *
      * @return Job[]
      */
-    protected function fetchJobs(): array
+    protected function fetchSchedules(): array
```

Comment:
I would add parameters to not call `params->get()` every time this method is called.